lib/fastlane/actions/hipchat.rb

```diff
@@ -18,11 +18,11 @@ def self.run(params)
 
         unless api_token
           Helper.log.fatal "Please add 'ENV[\"HIPCHAT_API_TOKEN\"] = \"your token\"' to your Fastfile's `before_all` section.".red
-          raise 'No HIPCHAT_API_TOKEN given.'.red
+          fail 'No HIPCHAT_API_TOKEN given.'.red
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.